pkgs/applications/science/logic/coq/default.nix

```diff
@@ -13,7 +13,8 @@
 , csdp ? null
 , version, coq-version ? null,
 }@args:
-let lib = import ../../../../build-support/coq/extra-lib.nix {inherit (stdenv) lib;}; in
+let lib' = lib; in
+let lib = import ../../../../build-support/coq/extra-lib.nix {lib = lib';}; in
```

Comment:
```suggestion
let lib = import ../../../../build-support/coq/extra-lib.nix {inherit lib;}; in
```

That does not work?

Reply:
Causes infinite recursion since we have an expression of the form `lib = ... lib ...`